dask/array/optimization.py

```diff
@@ -120,7 +120,7 @@ def hold_keys(dsk, dependencies):
 
 def _is_getter_task(
     value,
-) -> tuple[Callable, Any, Any, bool, bool | None] | None:
+) -> tuple[Callable, Any, Any, bool, bool | None] | None:  # type: ignore
```

Comment:
what was wrong here?

Reply:
I can't remember. I was having trouble getting it to work originally, but some of the other changes must have benefitted this section. Removing the ignore comments, mypy is satisfied with this section now.